.deployment/entrypoint.sh

```diff
@@ -15,6 +18,7 @@ currentSha="$(sha1sum ./package-lock.json)"
 if [ "$oldSha" != "$currentSha" ]; then
   echo "⚠️ Need to install packages due to updated package-lock.json"
   # When the checksums are not matching perform an npm install
+  rm -rf node_modules
```

Comment:
As far as I am aware - npm ci cleans the node_modules. I think we can remove this.

Reply:
it is not deleting the node_modules it is deleting the symlink as when ngcc has to recompile according to changes it causes troubles so deleting the symlink is necessary on github actions.